cfgov/v1/blocks.py

```diff
@@ -30,3 +33,39 @@ class Meta:
         handler = None
         method = 'POST'
         icon = 'form'
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False, label='ID for this content block', help_text=(
+            'Auto-generated on save, or enter some human-friendly text to make it easier to read.'
+            )
+        )
+
+    # def get_url():
+    #     return '/relative/path/'
+
+    # full_url = blocks.CharBlock(required=False, default=get_url(), help_text=(
+    #         'Copy this relative URL to insert anchor links that take the user to this content section.'
+    #         )
+    #     )
```

Comment:
Might be nice to remove these commented lines. The PR description can serve as future TODOs.